src/oscar/templates/oscar/dashboard/layout.html

```diff
@@ -121,6 +121,14 @@ <h1>{% block headertext %}{% endblock %}</h1>
 
 {% endblock %}
 
+{% block cdn_scripts %}
+  {{ block.super }}
+  {% if use_less and debug %}
```

Comment:
Please drop the `debug` check - we have provided enough warning about this, but should allow the user to test this without all the other debug components.

Reply:
I see. FYI debug is used on oscar/templates/oscar/layout.html too. Should I remove debug from there too? in a single commit?

Reply:
Oh, I had missed that, sorry. Let's leave as-is for now I think.